content/en/blog/_posts/2021-03-30-Kubernetes-Multi-Tenancy-Usage-Models.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: blog
+title: 'Three Common Tenancy Models For Kubernetes'
+date: 2021-03-30
+slug: kubernetes-multi-tenancy-usage-models
```

Comment:
BTW if you want to, we can update the slug to match the new title. I'm fine leaving it as is, mind.

Reply:
done!

Reply:
done!